app/assets/index.html

```diff
@@ -20,5 +20,6 @@
 </div>
 
 <script src="scripts/main.js"></script>
+<script>require('app/scripts/main');</script>
```

Comment:
I realized that a slightly simpler configuration would be to create two bundles (`main.js` and `modules.js`), where `modules.js` can even include the `main` module, but `main` is not `autoRequire`'d. This also has the advantage of making the `main` module testable.